index.html

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta name="color-scheme" content="light dark" />
-        <title>Adaptive Palette</title>
-    </head>
-    <body>
-        <div id="app"></div>
-    </body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="color-scheme" content="light dark" />
+    <link href="./src/PaletteCell.css" rel="stylesheet"/>
```

Comment:
1. I think it's agreed to use `scss` rather than `css` for styling. Scss is helpful in writing cleaner and more structural code.

2. The styling file doesn't need to be included in the html file. It can be directly imported into its component file. For example, to have `PaletteCell.css` take effect, in the `PaletteCell.ts`, write `import "./PaletteCell.scss"` in the import section.

Reply:
Thanks, and good to know.  I'll switch the `css` over to `scss` and `import` into the `.ts` file.